src/resolve/resolved-dns-packet.c

```diff
@@ -752,13 +752,24 @@ int dns_packet_append_opt(DnsPacket *p, uint16_t max_udp_size, bool edns0_do, in
                 static const uint8_t rfc6975[] = {
 
                         0, 5, /* OPTION_CODE: DAU */
+#if #if GCRYPT_VERSION_NUMBER >= 0x010600
```

Comment:
`#if #if`?

that's a typo!

(and tbh I would have added two `#if` conditions here, one around the length, and one around the new item we add... that way we can share the body of the list. This might not matter now, but sooner or later this will get annoying if we keep copying the list over and over again)

Reply:
You are right.